contracts/series/SeriesController.sol

```diff
@@ -1248,4 +1155,52 @@ contract SeriesController is
                 address(currentSeries.tokens.priceToken)
             );
     }
+
+    /// @notice This function allows the owner address to update allowed expirations for the auto series creation feature
+    /// @param timestamps timestamps to update
+    /// @dev Only the owner address should be allowed to call this
+    /// Expirations must be added in ascending order
+    /// Expirations must be aligned 8 AM weekly or daily
+    function updateAllowedExpirations(uint256[] calldata timestamps)
```

Comment:
Should this be `addAllowedExpirations` and `event AllowedExpirationAdded`? Otherwise a bit misleading, because it doesn't update previously added expirations, only adds new ones.

Reply:
I'll merge as is and we can decide later